src/ripple/app/paths/impl/PaymentView.h

```diff
@@ -39,12 +39,11 @@ namespace ripple {
     will change via the balanceHook and creditHook overrides
     of PaymentView.
 */
-class PaymentView : public View
+class PaymentView : public ViewWrapper<MetaView>
 {
 private:
-    MetaView view_;
     DeferredCredits tab_;
-    PaymentView* pv_ = nullptr;
+    PaymentView const* pv_ = nullptr;
```

Comment:
Clang release complains about unused var `pv_`. (The fix for deferred payments with stacked views does use this var, so the warning will go away when we merge that).